README.md

```diff
@@ -387,6 +387,13 @@ Will yield:
 
     Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the number of replicas associated with the metric that requires the largest number. For example, if WPA evaluates metric1, metric2, metric3, and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal is 30.
 
+- **Should I set replicas on my deployment?**
+
+    No. If you set the `replicas` value on your deployment, then every time you apply the deployment, the number of replicas will be reset to that value.
+    This means that if you have `replicas: 2` specified in your deployment but WPA has scaled up the deployment to be 10 pods, then when you apply the 
+    deployment the number of pods will quickly become 2 and you'll have to wait until WPA scales up again. 
+    To avoid this, omit setting `replicas` in your deployment. 
+    This ensures that there's no races between Kubernetes and WPA in setting the number of replicas correctly.
```

Comment:
```suggestion
    Omitting `replications` from your deployment ensures that there are no races between Kubernetes and WPA to set the number of replicas correctly.
```

Reply:
I can change this to "Omitting `replicas` from your deployment ..." but replications seems like its incorrect.